ios/Classes/ApproovHttpClientPlugin.m

```diff
@@ -472,7 +472,23 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
         [Approov setDevKey:call.arguments[@"devKey"]];
         result(nil);
     } else if ([@"getMessageSignature" isEqualToString:call.method]) {
-        result([Approov getMessageSignature:call.arguments[@"message"]]);
+        @try {
+            result([Approov getMessageSignature:call.arguments[@"message"]]);
+        }
+        @catch (NSException *exception) {
+            result([FlutterError errorWithCode:@"Approov.getMessageSignature"
+                                       message:exception.reason
+                                       details:nil]);
+        }
+    } else if ([@"getInstallMessageSignature" isEqualToString:call.method]) {
+        @try {
+            result([Approov getInstallMessageSignature:call.arguments[@"message"]]);
+        }
+        @catch (NSException *exception) {
```

Comment:
Why are these the only methods to catch exceptions? What happens if you don't catch the exception here?